src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala

```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```

Comment:
So you agree that if your file doesn't yet contain a copyright header but starts with some ScalaDoc, then your changes will delete the ScalaDoc?